packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNavItem.js

```diff
@@ -57,7 +57,7 @@ const styles = StyleSheet.create({
     color: colorGray300,
   },
   textSelected: {
-    color: colorBlue700,
+    color: colorSkyBlue,
```

Comment:
We should check if this blue should be Sky Blue and not another Blue

Reply:
This is what we have for Android currently, I'll check with Will.

Reply:
All good from Will :)

Reply:
Sweet! In which case will update my review!